app/Http/Controllers/UsersController.php

```diff
@@ -169,4 +170,24 @@ public function contestantsIndex()
 
         return view('users.contestants_index', compact('contestants'));
     }
+
+    /**
+     * Search for users.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function search(Request $request)
+    {
+        $query = $request->query('query');
+
+        // Redirect empty queries to the user index.
+        if ($query === '') {
+            return redirect()->route('users.index');
+        }
+
+        // Attempt to fetch all users.
+        $users = $this->manager->search($query);
+
+        return $users ? redirect()->route('users.show', [$users->first()->id]) : redirect()->route('users.index')->with('status', 'No user found!');
```

Comment:
hmm probably not. if we need to see a group of users we have other pages for that.